views/partials/top-level/security-tips.hbs

```diff
@@ -25,49 +20,52 @@
   </div>
 </main>
 <section class="container bg-light articles">
-  <div class="row jst-cntr">
+  <div class="row">
     <div class="col-9">
+      <section class="st-intro-wrap">
+        {{#each (getSecurityTipsIntro)}}
+          <p class="st-intro">{{{ this }}}</p>
```

Comment:
Another `{{{ }}}` usage, but `this` is our own hard-coded string here, and we've got https://github.com/mozilla/blurts-server/issues/724 to come back thru and audit these.